lint/lib/linter.js

```diff
@@ -68,11 +69,16 @@ class Linter {
 				if (svc.errorPath)
 					spawned[svc.errorPath].on("data", d => error += d.toString());
 				spawned.on("exit", () => {
-					resolve({
-						linter: svc.exe,
-						lintError: svc.processError ? svc.processError(error) : [],
-						result: svc.processResult(result)
-					});
+					try {
+						resolve({
+							linter: svc.exe,
+							lintError: svc.processError ? svc.processError(error) : [],
+							result: svc.processResult(result)
+						});
+					} catch (e) {
+						// Exception while parsing result or error
```

Comment:
I got Uncaught Exception sometimes because of empty output. But I didn't why.
Anyway, because parse error seems to stop later execution, I catch here and reject() it. :)